src/javascript/app_2/data/ws_methods.js

```diff
@@ -15,6 +15,15 @@ const WS = (() => {
     const getAccountStatus = () =>
         BinarySocket.send({ get_account_status: 1 });
 
+    const getOauthApps = () =>
```

Comment:
To be consistent, could you please remove extra `get` from functions name?
(Function names are now similar to the corresponding API method name. i.e. `oauthApps`, `portfolio` except the streaming ones which have `subscribe` prefix)